src/testUtils.js

```diff
@@ -0,0 +1,74 @@
+/**
+ *  Copyright (c) 2016-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* global expect, define, it */
+
+import fs from 'fs';
+import path from 'path';
+
+import jscodeshift from './core';
+
+/**
+ * Utility function to run a jscodeshift script within a unit test. This makes
+ * several assumptions about the environment:
+ *
+ * - `dirName` contains the name of the directory the test is located in. This
+ *   should normally be passed via __dirname.
+ * - The test should be located in a subdirectory next to the transform itself.
+ *   Commonly tests are located in a directory called __tests__.
+ * - `transformName` contains the filename of the transform being tested,
+ *   excluding the .js extension.
+ * - `testFilePrefix` optionally contains the name of the file with the test
+ *   data. If not specified, it defaults to the same value as `transformName`.
+ *   This will be suffixed with "-input" for the input file and "-output" for
+ *   the expected output. For example, if set to "foo", we will read the
+ *   "foo-input" file, pass this to the transform, and expect its output to
+ *   be equal to the contents of "foo-output".
+ * - Test data should be located in the same directory as the test file.
+ */
+export function runTest(dirName, transformName, options, testFilePrefix) {
+  if (!testFilePrefix) {
+    testFilePrefix = transformName;
+  }
+
+  const inputPath = path.join(dirName, testFilePrefix + '-input');
+  const source = fs.readFileSync(inputPath, 'utf8');
+  const expectedOutput = fs.readFileSync(
+    path.join(dirName, testFilePrefix + '-output'),
+    'utf8'
+  );
+  // Assumes transform is one level up from __tests__ directory
+  let transform = require(path.join(dirName, '..', transformName + '.js'));
+  // Handle ES6 modules using default export for the transform
+  if (transform.default) {
+    transform = transform.default;
+  }
+
+  const output = transform(
+    {path: inputPath, source},
+    {jscodeshift},
+    options || {}
+  );
+  expect((output || '').trim()).toEqual(expectedOutput.trim());
+}
+
+/**
+ * Handles some boilerplate around defining a simple jest/Jasmine test for a
+ * jscodeshift transform.
+ */
+export function defineTest(dirName, transformName, options, testFilePrefix) {
```

Comment:
I wanted to differentiate between defining a test (calling `define` and `it`) and running a test (actually running the transform and checking the result), thus the explicit function naming.